src/locales/fr.json

```diff
@@ -267,6 +268,19 @@
     "cancel": "Annuler",
     "error": "Je ne parviens pas à révoquer l’appareil, essayez de recharger la page."
   },
+  "configureDevice": {
+    "load_error": "Une erreur s’est produite pendant le chargement de vos dossiers, merci de réessayer prochainement.",
+    "configure_error": "Je ne parviens pas à configurer les dossiers à synchroniser avec votre ordinateur, essayez de recharger la page.",
+    "description": "Sélectionner les dossiers à synchroniser avec votre ordinateur :",
+    "no_folders": "Vous n'avez aucun dossier dans votre Cozy.",
+    "my_cozy": "Mon Cozy",
```

Comment:
:seedling: I should definitively check if unbreakable spaces are used in cozy-pass branding